$MirOS: ports/net/djbdns/patches/patch-man_axfrdns-conf_8,v 1.1 2006/01/17 22:40:38 tg Exp $
--- man/axfrdns-conf.8.orig	Thu Oct 23 08:09:45 2003
+++ man/axfrdns-conf.8	Tue Jan 24 21:49:19 2006
@@ -28,6 +28,18 @@ Normally 
 is 
 .IR /etc/axfrdns .
 
+.B axfrdns-conf
+enables
+.B axfrdns
+to run in two different ways.
+You can use
+.BR tcpserver (1)
+which is not included in the MirPorts Framework but the
+default mode of operation, or you can run it from within
+.BR inetd (8).
+The latter option however also forces you to run it outside
+of the control of djb daemontools.
+
 You can run the service under
 .BR svscan (8)
 by creating a symbolic link in the 
@@ -65,6 +77,29 @@ into
 .I tcp.cdb
 for
 .BR tcpserver (1).
+
+To run
+.B axfrdns
+within
+.BR inetd (8),
+first shut down the services after creating the
+directory, using the command:
+
+svc -dx 
+.I D
+.IR D /log
+
+After shutting down, remove everything except
+.IR D /env
+and
+.IR D /inetd
+to avoid them being picked up by
+.BR svscanboot (8)
+on next reboot (or don't install into /service at all).
+Finally, read the comment in
+.IR D /inetd
+and patch
+.BR inetd.conf (5).
 
 .B axfrdns-conf
 arranges for
